src/Apps/Search/Components/GenericSearchResultItem.tsx

```diff
@@ -0,0 +1,47 @@
+import { Flex, Image, Link, Sans, Serif, Spacer } from "@artsy/palette"
+import React, { SFC } from "react"
+
+interface GenericSearchResultItemProps {
+  item: any
+  index: number
+}
+
+export const GenericSearchResultItem: SFC<
```

Comment:
This should be of type `React.FC` -- `SFC` has been deprecated due to react hooks.